File: Baekjoon/Class2/02.30802.py
import math
import sys
input = sys.stdin.readline

n = int(input())
t_counts = list(map(int, input().split()))
t_bundle, p_bundle = map(int, input().split())

#티셔츠 묶음
t = 0
for t_count in t_counts:
    t += math.ceil(t_count / t_bundle)
    #t += (t_count + t_bundle - 1) // t_bundle

#펜 묶음과 낱개
p = n // p_bundle
p_rest = n % p_bundle

print(f"{t}\n{p} {p_rest}")

File: Baekjoon/Class2/02.30802.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim().split("\n");

let cursor = 0;
const n = Number(input[cursor++]);
const tCounts = input[cursor++].split(" ").map(Number);
const [tBundle, pBundle] = input[cursor++].split(" ").map(Number);

// 티셔츠 묶음
let t = 0;
for(const tCount of tCounts)
    t += Math.ceil(tCount / tBundle);
    // t += Math.trunc((tCount + tBundle - 1) / tBundle);

// 펜 묶음과 낱개
const p = Math.trunc(n / pBundle);
const pRest = n % pBundle;

console.log(`${t}\n${p} ${pRest}`);